tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Christoffel.cpp

```diff
@@ -39,4 +59,102 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.Christoffel.",
   test_christoffel<1, IndexType::Spacetime>(0.);
   test_christoffel<2, IndexType::Spacetime>(0.);
   test_christoffel<3, IndexType::Spacetime>(0.);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(gr::Tags::SpatialChristoffelFirstKind<3, Frame::Inertial,
+                                              DataVector>::name() ==
+        "SpatialChristoffelFirstKind");
+  CHECK(gr::Tags::TraceSpatialChristoffelFirstKind<3, Frame::Inertial,
+                                                   DataVector>::name() ==
+        "TraceSpatialChristoffelFirstKind");
+  CHECK(gr::Tags::SpatialChristoffelSecondKind<3, Frame::Inertial,
+                                               DataVector>::name() ==
+        "SpatialChristoffelSecondKind");
+  CHECK(gr::Tags::TraceSpatialChristoffelSecondKind<3, Frame::Inertial,
+                                                    DataVector>::name() ==
+        "TraceSpatialChristoffelSecondKind");
+
+  CHECK(gr::Tags::SpatialChristoffelFirstKindCompute<3, Frame::Inertial,
+                                                     DataVector>::name() ==
+        "SpatialChristoffelFirstKind");
+  CHECK(gr::Tags::TraceSpatialChristoffelFirstKindCompute<3, Frame::Inertial,
+                                                          DataVector>::name() ==
+        "TraceSpatialChristoffelFirstKind");
+  CHECK(gr::Tags::SpatialChristoffelSecondKindCompute<3, Frame::Inertial,
+                                                      DataVector>::name() ==
+        "SpatialChristoffelSecondKind");
+
+  // Check that the compute items return correct values
+  const DataVector test_vector{5., 4.};
+  auto spacetime_metric =
+      make_with_value<tnsr::aa<DataVector, 3, Frame::Inertial>>(test_vector,
+                                                                0.);
+  get<0, 0>(spacetime_metric) = -1.5;
```

Comment:
given that you're not using the explicit values to manually compute an expected value, it seems both more succinct and more thorough to just generate random values using either `fill_with_random_values` or `make_with_random_values`. If you do the latter, you might also be able to mark the inputs `const`.

Reply:
agreed & changed